Cargo.toml

```diff
@@ -72,6 +72,7 @@ commander = { path = "util/commander" }
 suzuka-config = { path = "networks/suzuka/suzuka-config" }
 monza-config = { path = "networks/monza/monza-config" }
 # util
+flocks = { path = "util/flocks" }
```

Comment:
Can we just names that are more self descriptive, `file-locks` is immediedtly understandable `flocks`, one has to remember what it means. To a new developer they would have to check or look up a glossary.

Reply:
`flocks` is the Unix term though. But, I suppose this is more more general. If @mzabaluev votes your way, then I'll change it.